spec/models/maneuver_participant_spec.rb

```diff
@@ -1,4 +1,32 @@
 require 'rails_helper'
-RSpec.describe ManeuverParticipant, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe ManeuverParticipant do
+  describe 'score' do
+    it'score should be 22' do
+      maneuver = create :maneuver_participant,
+                        speed_achieved: true, obstacles_hit: { 1 => 2, 2 => 1 },
+                        distances_achieved: { [1, 2] => 3 }
+      expect(maneuver.score).to be(22)
+    end
+  end
+
+  describe 'previous participant' do
+    it'finds previous participant' do
+      partip_1 = create :maneuver_participant
+      partip_2 = create :maneuver_participant
+      my_maneuver = create :maneuver,
```

Comment:
In particular, there really isn't a me, my, or I anywhere in idiomatic Rails code